helm/ffc-demo-web/templates/config-map.yaml

```diff
@@ -0,0 +1,24 @@
+{{- include "ffc-helm-library.container-config-map" (list . "ffc-demo-web.config-map") -}}
+{{- define "ffc-demo-web.config-map" -}}
+data:
+  NODE_ENV: {{ quote .Values.environment }}
+  PORT: {{ quote .Values.container.port }}
+  COOKIE_PASSWORD: {{ quote .Values.container.cookiePassword }}
+  OKTA_DOMAIN: {{ quote .Values.okta.domain }}
+  OKTA_CLIENT_ID: {{ quote .Values.okta.clientId }}
+  OKTA_CLIENT_SECRET: {{ quote .Values.okta.clientSecret }}
+  OKTA_AUTH_SERVER_ID: {{ quote .Values.okta.authorizationServerId }}  
+{{- if .Values.pr }}
+   SITE_URL: "{{ .Values.container.siteUrl }}-{{ .Values.pr }}.{{ .Values.ingress.server }}"
```

Comment:
Remove the extra whitespace before SITE_URL, all indentation needs to be the same